gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-kafka/src/main/java/io/gravitee/plugin/endpoint/kafka/KafkaEndpointConnector.java

```diff
@@ -98,10 +100,10 @@ protected Completable publish(final ExecutionContext ctx) {
                     // Set kafka producer properties
                     config.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
                     config.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, ByteArraySerializer.class);
-                    config.put(ProducerConfig.CLIENT_ID_CONFIG, UUID.randomUUID().toString());
+                    config.put(ProducerConfig.CLIENT_ID_CONFIG, KAFKA_CLIENT_IDENTIFIER);
```

Comment:
Regarding the documentation, I'm still not sure to have a good understanding of what we should consider to be a "Kakfa client id". What if, for the same api and connector instance, we have different configuration because of the QoS. It will result in 2 different producer instances with the same Kafka client identifier, it is allowed to do that 🧐 ?

Reply:
Ok, given that I'm wondering if putting a random UUID will really help to identify "who" published something ? 
I mean, if the objective is to offer a better traceability, should we compose a custom identifier with information that could help to identify that the origin is gravitee apim ? Could be `gio-apim-<api-name>-<tiny-uuid>` for instance ? Also, regarding performances, do we know the additional cost of adding this id (is it transmitted on each message ? We should opt for a short id if so).